providers/base/units/stress/jobs.pxu

```diff
@@ -680,3 +680,52 @@ command:
 environ: WPA_N_SSID WPA_N_PSK
 category_id: com.canonical.plainbox::stress
 flags: preserve-locale also-after-suspend
+
```

Comment:
Perhaps I didn't make it clear enough with my previous review, but I wish these jobs (`stress/store_and_change_oomd_config` and `stress/restore_oomd_config`) can be moved to stress/stress-ng.pxu

Reply:
Copy that. Done.